extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md

```diff
@@ -66,12 +66,22 @@ Python interpreter.
 
     import nest
 
+
+It should be noted, however, that certain external packages must be imported *before* importing nest. 
+These include `sklearn` ([scikit-learn](http://scikit-learn.org/stable/index.html)) and `scipy` ([SciPy](https://www.scipy.org/)):
+
+    from sklearn.svm import LinearSVC
```

Comment:
I can confirm the sklearn bug on Python 3.6.4 and SKL 0.19.1, but not with scipy.
Also importing only ``sklearn`` and not the specific submodule does not solve the problem.

Reply:
@Silmathoron thanks for double-checking. In this case I agree with the suggested changes.